tests/model_inheritance/test_abstract_inheritance.py

```diff
@@ -447,3 +447,34 @@ def fields(model):
                 ("name", models.CharField),
             ],
         )
+
+    @isolate_apps('model_inheritance')
+    def test_permission_inheritance_abstract(self):
+        """
+        Abstract permissions with %(model_name)s interpolation should become concrete
+        """
+
+        class A(models.Model):
+            class Meta:
+                app_label = 'model_inheritance'
+                abstract = True
+                permissions = [
+                    ('check_%(model_name)s', 'Check %(verbose_name)s'),
```

Comment:
We should add a custom `verbose_name` to ensure that `model_name` is not interpolated here.